CHANGES.txt

```diff
@@ -4,14 +4,33 @@ Changelog
 2.1.0 (unreleased)
 ------------------
 
+Announcements:
+
+* Change all references to the pep8 project to say pycodestyle; #530
+
 Changes:
 
-* Added check E74x for using variables named 'l', 'O', or 'I'; #341
-* Improved performance of `compound_statements` check; #314 / #522
-* Fixed remaining references to `pep8`; #518 / #530
-* Added `noqa` support for `maximum_line_length` check; #538
-* Added check E305 for two blank lines after toplevel methods and classes; #400
+* Add Gitter badge; #557
+* Report E302 for blank lines before an "async def"; #556
+* Updated tox to do same tests as Travis CI (`setup.py test`).
+* Updated to use sheilds.io badges.
+* Updated text about running tests with tox; #552
+* Update maximum_line_length to use Checker.noqa; 538
+* Updated Travis CI doc link.
+* Updated docs related to tested Python versions.
+* Added skip missing interpreters flag to tox config.
+* Add test-requirements.txt with basic test packages.
+* Report E742 and E743 for badly named functions and classes.
+* Also report E741 on 'global' and 'nonlocal' statements.
+* Report E741 for prohibited single-letter variables.
+* Added venv/ (virtualenv) to list of git ignored filename patterns.
+
+Bugs:
 
+* Stop checking for string option type; #561
+* Require two blank lines after toplevel def, class; #536
+* Badly named single-letter variable names are not detected; #341
+* Fixing compound_statement not to be quadratic in # of :s; #314
```

Comment:
Can we reword this a bit, e.g., 

```
Remove accidentally quadratic computation based on the number of colons. This will make pycodestyle faster in some cases.
```